### ISSUE TITLE
FTLD IVP blanking rule throws an error, for fields FTDFDMAM, FTDAMDID

### ISSUE THREAD

**Issue by elijahrockers** [ctsit/nacculator#120]
```
Traceback (most recent call last):
  File "/usr/local/bin/redcap2nacc", line 11, in <module>
    load_entry_point('nacculator==1.7.0', 'console_scripts', 'redcap2nacc')()
  File "/usr/local/lib/python3.6/site-packages/nacculator-1.7.0-py3.6.egg/nacc/redcap2nacc.py", line 559, in main
  File "/usr/local/lib/python3.6/site-packages/nacculator-1.7.0-py3.6.egg/nacc/redcap2nacc.py", line 425, in convert
  File "/usr/local/lib/python3.6/site-packages/nacculator-1.7.0-py3.6.egg/nacc/redcap2nacc.py", line 71, in check_blanks
  File "/usr/local/lib/python3.6/site-packages/nacculator-1.7.0-py3.6.egg/nacc/ftld/blanks.py", line 124, in convert_rule_to_python
Exception: Could not parse Blanking rule: FTDFDMAM
Traceback (most recent call last):
  File "/usr/local/bin/redcap2nacc", line 11, in <module>
    load_entry_point('nacculator==1.7.0', 'console_scripts', 'redcap2nacc')()
  File "/usr/local/lib/python3.6/site-packages/nacculator-1.7.0-py3.6.egg/nacc/redcap2nacc.py", line 559, in main
  File "/usr/local/lib/python3.6/site-packages/nacculator-1.7.0-py3.6.egg/nacc/redcap2nacc.py", line 425, in convert
  File "/usr/local/lib/python3.6/site-packages/nacculator-1.7.0-py3.6.egg/nacc/redcap2nacc.py", line 71, in check_blanks
  File "/usr/local/lib/python3.6/site-packages/nacculator-1.7.0-py3.6.egg/nacc/ftld/blanks.py", line 124, in convert_rule_to_python
Exception: Could not parse Blanking rule: FTDAMDID
```
The blanking code looks a little wild for me, I'm not sure why it would be throwing an error, and not sure how to fix it.

**Comment by elijahrockers** [comment]
The form definitions for these are incorrect. They include commas, which is breaking the ability of ''convert_rule_to_python'' to correctly interpret the rules. Removing the commas from the ftld/ivp/forms.py rule definitions for these fields fixes the issue.

This mostly appears to affect the FTLD module. I will make a commit to the current pull request feature/date_rater_fix to try to catch as many of these as possible.

**Comment by elijahrockers** [comment]
Should be fixed now by #118

**Comment by s-emerson** [comment]
I think I finally addressed this issue with the recent hotfix 1.8.2. I updated the FTLD module's blanks.py file to be able to handle extra commas. Thank you for making a note.
